### PR TITLE
guiscrcpy: add version 2022.7.1

### LINKED ISSUE #8818: [Request] guiscrcpy
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** [name of new package]
GuiScrcpy
**Description:** [clear and concise details of what it is]
A full fledged GUI integration for the award winning open-source android screen mirroring system -- scrcpy
**Homepage:** [a URL/link]
https://guiscrcpy.srev.in/
**Download link(s):** [URL(s)/link(s)]
https://github.com/srevinsaju/guiscrcpy/releases

**Some indication of popularity/repute:** [GitHub stars/software reviews etc.]

1.7k stars

### PR DESCRIPTION
**Pull request by Lutra-Fs**
Closes #8818 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/guiscrcpy.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "##": "User config is stored at '$Env:LocalAppData\\guiscrcpy'",
+    "version": "2022.7.1",
+    "description": "A simple, pluggable, graphical user interface for the fastest Android screen mirroring software, scrcpy",
+    "homepage": "https://github.com/srevinsaju/guiscrcpy",
+    "license": "GPL-3.0-or-later",
+    "suggest": {
```
Comment (niheaven) [inline bucket/guiscrcpy.json:7]: A depends?

**Review by niheaven — APPROVED** [review]

**Comment by Lutra-Fs** [comment]
Is pyqt5 version okay shipped with Scoop? It currently offered 2 versions pyqt5 and pyside2

**Comment by Lutra-Fs** [comment]
See https://github.com/srevinsaju/guiscrcpy/releases/tag/v4.7.2 regrading on 2 versions' difference,

**Comment by niheaven** [comment]
It seems like PySide2 version is smaller? Using both of them is okay, and maybe a smaller one is better.

File: bucket/guiscrcpy.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "##": "User config is stored at '$Env:LocalAppData\\guiscrcpy'",
+    "version": "2022.7.1",
+    "description": "A simple, pluggable, graphical user interface for the fastest Android screen mirroring software, scrcpy",
+    "homepage": "https://github.com/srevinsaju/guiscrcpy",
+    "license": "GPL-3.0-or-later",
+    "depends": "scrcpy",
+    "notes": "If you are required to specify scrcpy-server's location, you can find it in your scrcpy installation directory.",
```
Comment (tech189) [inline bucket/guiscrcpy.json:8]:
```suggestion
    "notes": "If you are required to specify scrcpy-server's location, you can find it in $dir.",
```
Expands to the full path of the installation directory - more helpful for users.

Comment (Lutra-Fs) [reply bucket/guiscrcpy.json:8]: $dir will return guiscrcpy’s installation path but this is about scrcpy’s

Comment (tech189) [reply bucket/guiscrcpy.json:8]: Sadly we can't expand Scoop's installation path in notes. But we could instead put: 
```suggestion
    "notes": "If you are required to specify scrcpy-server's location, you can find it in ~\scoop\apps\scrcpy\current\",
```
Those who installed Scoop globally will understand where to find it from this abbreviated path.

**Review by tech189 — APPROVED** [review]

**Pull request merged**